src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -621,6 +621,7 @@ namespace Akka.Actor
         public static readonly string NoDispatcherGiven;
         public static readonly string NoMailboxGiven;
         public static readonly Akka.Actor.Scope NoScopeGiven;
+        public const int NoStashSize = -1;
```

Comment:
This what `IStash.Capacity` has historically defaulted to when working with unbounded stashes - all I did was turn it into a constant.